MISRA.md

```diff
@@ -31,6 +31,18 @@ _Ref 8.4.1_
         a declaration in header file is not useful as the assembly code will
         still need to declare it separately.
 
+#### Rule 21.6
+
+_Ref 21.6.1_
+
+- MISRA C-2012 Rule 21.6: The Standard Library input/output functions shall not
+        be used.
+        This rule warns about the use of standard library input/output functions
+        as they might have implementation defined or undefined behavior. The
+        function `snprintf` is used for convenience only when `configUSE_TRACE_FACILITY`
```

Comment:
We can change the deviation statement as:
The function 'snprintf' is used for debugging only ( when `configUSE_TRACE_FACILITY`  is set to 1 and `configUSE_STATS_FORMATTING_FUNCTIONS` is set to greater than 0 ) and is not part of the 'core' kernel code.

Reply:
Thank you for the suggestion updated in new commit.